packages/@n8n/task-runner-node-js/src/task-runner.ts

```diff
@@ -102,8 +100,8 @@ export class TaskRunner {
 
 	deleteStaleOffers() {
 		for (const key of Object.keys(this.openOffers)) {
```

Comment:
nit: I think this can be done without using `Object.keys`
```suggestion
		this.openOffers.forEach((offer, key) => {
		  if (offer.validUntil < process.hrtime.bigint()) {
				this.openOffers.delete(key);
			}
		})
```